mtp_api/apps/transaction/models.py

```diff
@@ -22,16 +23,20 @@ class Transaction(TimeStampedModel):
     prisoner_dob = models.DateField(blank=True, null=True)
 
     amount = models.PositiveIntegerField()
-    sender_bank_reference = models.CharField(
-        blank=True, max_length=250
-    )
-    sender_customer_reference = models.CharField(
-        blank=True, max_length=250
-    )
+
+    # cannot be empty otherwise we can't send the money back
+    sender_sort_code = models.CharField(max_length=50)
+    sender_account_nr = models.CharField(max_length=50)
+    sender_name = models.CharField(max_length=100)
```

Comment:
Is the `sender_name` length from the BAI2 spec? Might be worth considering a longer field length, or making a text field because some people might have crazy long names…